tool.go

```diff
@@ -143,7 +143,8 @@ func main() {
 					}
 					if len(pkgs) == 1 { // Only consider writing output if single package specified.
 						if pkgObj == "" {
-							if pkg.ImportPath == "." {
+							importPath := filepath.Base(pkg.ImportPath)
+							if importPath == "." || importPath == ".." {
 								pkgObj = filepath.Base(currentDirectory) + ".js"
```

Comment:
I don't think this is right. I think for the `..` case, we want to use the name of the parent directory.

Reply:
Thanks for pointing that out.